userguide/image_similarity/README.md

```diff
@@ -111,6 +111,68 @@ similar_images = similarity_graph.edges
 [91440 rows x 4 columns]
 ```
 
+##### Deploying to Core ML
+
+With the Core ML framework, you can use the model to
```

Comment:
How about this for the first sentence? `Using the Core ML Framework, you can retrieve the distance between a query image and all images in the reference data.`